.github/workflows/pull_request.yaml

```diff
@@ -18,9 +18,15 @@ on:
         type: boolean
         required: false
         default: false
+      runHelmLint:
+        description: "Flag to enable helm lint"
+        type: boolean
+        required: false
+        default: false
```

Comment:
```suggestion
        default: true
```

Reply:
@shimoncohen actually i want it to be false so they devs won't be blocked by default and they'll have the choice whether to enable it

Reply:
@shimoncohen done